### ISSUE TITLE
HeapDumpOnOutOfMemory

### ISSUE THREAD

**Issue by tes5884** [jitsi/jicofo#284]
I'm seeing a bunch of these errors when looking at htop. The issue is that my resources aren't being used at all. Is there a way to specify how much memory Jicofo should use?

**Comment by mdsystems** [comment]
I am commenting on this old issue, because I am trying to troubleshoot a Java process consuming 100% of one of my VCPUs. I think what you are seeing in htop is just the full command line used to run a Java process and though I am not a Jitsi expert (learning about it at the moment), I think it is probably to be expected.

**Comment by mdsystems** [comment]
@damencho When I have more info, I will post on the community forum, rather than here. I only posted a comment on this thread to partially answer tes5884's question. 

To answer your question, in my situation it is a single Java process owned by the JVB user but I have no more info at present. It does not always happen, but once it starts, it persists, even when no calls are active. I'll post and more detail once I have it.

**Comment by damencho** [comment]
@mdsystems I had posted in the forum several times about disabling the TCP harvester, and apparently it was fixing the issue for people. We are planing disabling that in general ...

**Comment by mdsystems** [comment]
Actually, I was here posting a response to someone's question, which had been ignored for over a year because I observed the same thing. I thought that this issue was a "non-issue" and commented accordingly, in case anyone else came here having noted what they thought was a problem, when it is not. Please read my initial comment again - I was emphatically not posting about an issue that I found.

You kindly suggested that I post in the forums with regard to an issue that I was having, which I confirmed I would do. You'll note that I was not reporting an issue here, simply because I was not sure what the issue was.

**Comment by damencho** [comment]
Yep, understood and sorry if I sound bad and offending you in any way, I didn't mean anything like that, I just wanted to explain something and apparently, I misread and misunderstood you initially :)

**Comment by mdsystems** [comment]
Not a problem and no offence caused at all. I do understand the need to keep the repository free from support issues and was trying to do my bit (for what it's worth) to comment where I felt it was useful. Thank you for your suggestion regarding the TCP harvester. I am checking that out now.
